src/main/java/com/google/api/codegen/grpcmetadatagen/GrpcMetadataProviderFactory.java

```diff
@@ -50,14 +50,16 @@
             new JavaPackageMetadataTransformer(
                 ImmutableMap.of(
                     "LICENSE.snip", "LICENSE",
-                    "metadatagen/java/grpc/build_grpc.gradle.snip", "build.gradle"),
+                    "metadatagen/java/grpc/build_grpc.gradle.snip", "build.gradle",
+                    "metadatagen/java/grpc/pom_grpc.xml.snip", "pom.xml"),
```

Comment:
Why generate both gradle and maven files?

Reply:
I mentioned it in the PR description: "(oping the pom.xml out of self-service is TBD)". We should not do it, but it seems it does not hurt (self-serivce workflow does not even imply people investigating the proto/grpc generated dirs, just run build from the root. Also before the recent cleanup we had a lot of random garbage in proto/gprc generated folders, so our customers are used to see random stuff there.

So this is a small issue, which shoudl be addressed, but after the bigger change (move to gcj).

Reply:
Ok, I'll allow it given impending work on codegen/packagegen splitting.